compiler/rustc_hir_typeck/src/closure.rs

```diff
@@ -396,6 +396,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     /// with_closure(|x: &u32| { ... })
     /// ```
     ///
+    // ignore-tidy-odd-backticks FIXME complete the unmatched backtick below
     /// Here:
     /// - E would be `fn(&u32) -> &u32`.
     /// - S would be `fn(&u32) ->
```

Comment:
```suggestion
    /// - S would be `fn(&u32) -> ?T`
```
To complete the FIXME.

Reply:
Thanks! I have tried studying 053383dbeff008744fb5dea7c1c8f4fba98cc06e as well as the PR that came with it, but in the end didn't want to do something wrongly. There seems to be some transition going on but I have no clue what it is about.

Reply:
I am not _entirely_ sure that this is correct, but given the rest of the comment it probably is. Also makes the most sense

You didn't remove the ignore though :)